src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -106,7 +113,7 @@ public void safeState() {
     @Override
     public void periodic() {
         super.periodic();
-
+        heightPub.set(getData().heightAbsInches / elevatorMaxHeight);
```

Comment:
The max elevator height should come from the hard limits in the map, not a constant in this subsystem file.